stories/webapp/accountdashboard/WordPressIntegrationStory.js

```diff
@@ -0,0 +1,105 @@
+        Expires on 30/04/2020
+      </span>
+    </Fragment>
+  );
+
+  return (
+    <ModalTrigger>
+      <Modal.Container>
+        <Modal.Header text={modalSubHeader}>WordPress demo</Modal.Header>
+        <Modal.Body className="has-columns">
+          <div className="modal__column bg-neutral-98">
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="preview-url"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Preview URL
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="preview-url"
+                  value="https://cms-wordpress.gatherpreview.com/acme"
+                  onClick={action('copy preview url')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+            <div className="border border-solid border-neutral-90 rounded bg-white mb-half p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="preview-admin-url"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Admin URL
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="preview-admin-url"
+                  value="https://cms-wordpress.gatherpreview.com/acme/wp-admin"
+                  onClick={action('copy admin url')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+            <div className="border border-solid border-neutral-90 rounded bg-white p-half flex items-center justify-between">
+              <label // eslint-disable-line jsx-a11y/label-has-associated-control
+                htmlFor="password"
+                className="w-1/4 font-semi-bold ml-half"
+              >
+                Password
+              </label>
+              <div className="flex-grow">
+                <InputWithButton
+                  inputId="password"
+                  value="P4$$W0RD!"
+                  onClick={action('copy password')}
+                  // onClick={copyElementValueToClipboard}
+                  buttonText="Copy link"
+                  buttonTextAfterClick="Copied!"
+                />
+              </div>
+            </div>
+          </div>
+        </Modal.Body>
+        <Modal.Footer>
+          <div className="flex-row-reverse p-quarter">
+            <Button types={['link']} clickHandler={action('clickHandler')}>
```

Comment:
🔴 `Button` should use `onClick`. `clickHandler` is debt and will be phased out 👍